source/replication/site-replication-overview.rst

```diff
@@ -0,0 +1,462 @@
+      
+         or define environment variables
+      
+         .. code-block:: shell
+         
+            export MC_HOST_minio1=https://adminuser:adminpassword@minio1.example.com
+            export MC_HOST_minio2=https://adminuser:adminpassword@minio2.example.com
+            export MC_HOST_minio3=https://adminuser:adminpassword@minio3.example.com
+      
+      #. Add site replication configuration
+
+         List all existing replicated sites first, then list the new site(s) to add.
+         In this example, ``minio1``, ``minio2``, and ``minio3`` are already configured for replication.
+         The command adds minio4 and minio5 as new sites to add to the replication.
+         ``minio4`` and ``minio5`` must be empty.
+      
+         .. code-block:: shell
+         
+            mc admin replicate add minio1 minio2 minio3 minio4 minio5
+      
+      #. Query the site replication configuration to verify
+      
+         .. code-block:: shell
+         
+            mc admin replicate info minio1
+
+Modify a site's endpoint
+------------------------
+
+From time to time a replicated site's address may change.
+When this occurs, update the site's endpoint in the site replication's configuration.
+
+.. tab-set::
+
+   .. tab-item:: Console
+
+      #. In a browser, access the Console for one of the replicated sites
+
+         For example, ``https://<addressforsite>:9000``
+
+      #. Select **Settings**, then **Site Replication**
+      
+      #. Select the pencil **Edit** icon to the side of the site to update
+
+         .. image:: /images/minio-console/console-site-replication-edit-button.png
+            :width: 600px
+            :alt: MinIO Console's List of Replicated Sites screen with the edit buttons highlighted
+            :align: center
+
+      #. Make the following entries:
+
+         :New Endpoint: `(required)` The new endpoint address and port to use.
+
+         .. image:: /images/minio-console/console-settings-site-replication-edit-endpoint.png
+            :width: 600px
+            :alt: Example of the MinIO Console's Edit Replication Endpoint screen
+            :align: center
+
+      #. Click **Update**
+
+   .. tab-item:: Command line
+
+      #. Obtain the site's Deployment ID with :mc-cmd:`mc admin replicate info`
+
+         .. code-block:: shell
+
+            mc admin replicate info <ALIAS>
+         
+      
+      #. Update the site's endpoint with :mc-cmd:`mc admin replicate edit`
+      
+         .. code-block:: shell
+
+            mc admin replicate edit ALIAS --deployment-id [DEPLOYMENT-ID] --endpoint [NEW-ENDPOINT]
+
+         Replace [DEPLOYMENT-ID] with the deployment ID of the site to update.
+         
+         Replace [NEW-ENDPOINT] with the new endpoint for the site.
+
+Remove a Site from replication
+------------------------------
+
+When decomissioning a peer site, you must remove it from the site replication configuration.
```

Comment:
Decommissioning here could be misconstrued with pool decommissioning. Maybe just "You can remove a site from the replication configuration at any time. You cannot re-add the site at a later date without first completely wiping data from that site and re-adding it."

Assuming the second part based on the whole deployment ID thing. Engineering can confirm.